src/tests/Common/ilasm/ilasm.ilproj

```diff
@@ -5,5 +5,6 @@
 
   <PropertyGroup>
     <RuntimeIdentifier>$(OutputRid)</RuntimeIdentifier>
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
```

Comment:
Disabling this warning does not sound right

Reply:
Test project want to use `Microsoft.NETCore.App.Runtime.linux-loongarch64` version `7.0.0` which is not exists. Since I use locally built version it produces `8.0.0-dev` and because I'm early in the cycle I "hope" that it's not important yet and while I make things working, I can remove this and use separate feed/folder with proper versions.

Test project which as for package - src/tests/Common/test_dependencies_fs/test_dependencies.fsproj

Reply:
@kant2002 I think this shouldn't be necessary anymore since NetCoreAppToolCurrent which this project is targetting is set to `net8.0` instead of `net7.0` now.

Reply:
Removed warning. Let's see how it worked out from main after this merges.